src/session-pool.js

```diff
@@ -112,11 +113,26 @@ function SessionPool(database, options) {
   this.evictHandle_ = null;
   this.pingHandle_ = null;
 
+  this.traces_ = new Map();
+
   EventEmitter.call(this);
 }
 
 util.inherits(SessionPool, EventEmitter);
 
+/**
+ *
+ */
+SessionPool.formatTrace_ = function(trace) {
+  var formatted = trace.slice(2).map(function(t) {
+    return `  at ${
```

Comment:
I checked a native-thrown error, and it uses 4 spaces:

```js
$ node -e "throw new Error('no')"
[eval]:1
throw new Error('no')
^

Error: no
    at [eval]:1:7
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at Object.runInThisContext (vm.js:152:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:641:30)
    at evalScript (bootstrap_node.js:470:27)
    at startup (bootstrap_node.js:167:9)
    at bootstrap_node.js:613:3
```

Not sure if that's just for me / my terminal / my Node version, etc. Can you see what you get If you get the same, and then if you do, let's follow suit.

Reply:
SGTM